generate/seccomp/doc.go

```diff
@@ -1,2 +0,0 @@
-// Package seccomp includes functions for configuring seccomp.
-package seccomp
```

Comment:
On Thu, Jul 28, 2016 at 08:19:16AM -0700, Grant Seltzer Richman wrote:
> > @@ -1,2 +0,0 @@
> > -// Package seccomp includes functions for configuring seccomp.
> > -package seccomp
> 
> Should we add it back later when it's more complete? 

I'm fine filling it in now, if we have more to say.  But if we don't
have more to say I'd rather leave it alone.  Do we gain something by
removing it?